### ISSUE TITLE
Include tag aerialway for foot/bike?

### ISSUE THREAD

**Issue by karussell** [graphhopper/graphhopper#344]
[Example](http://www.openstreetmap.org/directions?engine=graphhopper_foot&route=43.6720%2C40.2976%3B43.6248%2C40.3106#map=14/43.6524/40.3167)

**Comment by matkoniecz** [comment]
Personally I would prefer to be routed in my hiking/cycling route just on paths.

**Comment by karussell** [comment]
Sure, you can customize this to your needs. But here we need to support general 'by-bike' or 'by-foot'

**Comment by scaidermern** [comment]
> Personally I would prefer to be routed in my hiking/cycling route just on paths.

And some people would prefer being routed by car just on roads but not ferries. Yet they have to be supported for reaching certain regions.

In my opinion aerialways should be supported by adding a reasonable penalty. If the user still doesn't like to get routed via an aerialway he/she still can set a corresponding via-point to avoid it.

**Comment by agsochi** [comment]
There is no alternative route to aerial ways in example region.

**Comment by qeith** [comment]
I have to agree that aerialways be supported with a sensible penalty.  In many regions they are the only way to cross rivers.  This is true in the mountains of India, Pakistan, and Nepal -- but I suspect other developing countries too. [Here](http://www.openstreetmap.org/directions?engine=graphhopper_foot&route=34.00846%2C77.20785%3B34.00890%2C77.20662#map=18/34.00874/77.20821) is an example.  In these regions simple cable cars pulled by hand are often built as a less expensive option to bridges.  As my example, failure to support these aerialways results in serious misrouting.

**Comment by karussell** [comment]
Yes, please provide a pull request which we can review and discuss. I agree that aerialway should be added but I would need time to dig into it and make it properly. Also I do not have a realworld example in mind which I could check if afterwards everything is reasonable :)

**Comment by pyrog** [comment]
Example in France, the [Téléo](https://www.teleo-tisseo.fr/) of Toulouse ([osm](https://www.openstreetmap.org/way/749366976)).

**Comment by georgbachmann** [comment]
This also came up in our team lately. The option as such would be nice.
But I thing cheking OSM-Data would be needed? I found some aerialways that don't  have a proper connection to the rest of the road-network
Like this one here for example: https://www.openstreetmap.org/#map=19/47.66605/11.89004
But as I understand it, Graphhopper needs that, right?

**Comment by scaidermern** [comment]
> I found some aerialways that don't have a proper connection to the rest of the road-network Like this one here for example: https://www.openstreetmap.org/#map=19/47.66605/11.89004 But as I understand it, Graphhopper needs that, right?

Yes, a proper connection is needed for routing. Please add one (maybe `highway=footway` + `footway=link`?). To start with, the geometry does not have to match reality exactly, as it can be improved later (but of course you should try to add to it in a meaningful way).

**Comment by georgbachmann** [comment]
Ok... I noticed that on quite a few aerialways and just wanted to point it out as it might be problematic for Graphhopper. But yes.. I will fix some in my region... But I guess there will be a ton of those in OSM...
